### PR TITLE
Hearing Worksheet | Deleting warning confirmation 

### PR DESCRIPTION
**Pull request by nolawi**
Issue #3280

- [ ] Adds a confirmable modal 
- [ ] adds user clicks on the trashcan to delete an issue. 
- [ ] Added new styleguide issue for bg scroll. 

~~- [ ]  Triage with design about highlighting row on hover since modal bg hides it~~

### CODE REVIEW

File: app/assets/stylesheets/_hearings.scss [hunk 1]
```diff
@@ -309,6 +309,24 @@ $color-hearings-saving: #777;
...
   }
 
+  .cf-modal {
+
+    h2 {
+      border-bottom: none;
+      font-size: 1.5em;
+    }
+
+    .cf-modal-divider {
```
Comment (mmlumba) [inline app/assets/stylesheets/_hearings.scss]: There is a prop on the modal I added to actually handle this: https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/components/Modal.jsx#L79

Comment (nolawi) [reply app/assets/stylesheets/_hearings.scss]: cool i will update..

File: client/app/hearings/components/HearingWorksheetIssues.jsx [hunk 2]
```diff
@@ -4,21 +4,40 @@ import PropTypes from 'prop-types';
...
 
 import { TrashCan } from '../../components/RenderFunctions';
 
 class HearingWorksheetIssues extends PureComponent {
 
-  getKeyForRow = (index) => {
-    return index;
+  constructor(props) {
+    super(props);
+    window.jqueryOn = false;
```
Comment (sharonwarner) [inline client/app/hearings/components/HearingWorksheetIssues.jsx]: What does this line do?

File: client/app/hearings/components/HearingWorksheetIssues.jsx [hunk 3]
```diff
@@ -4,21 +4,40 @@ import PropTypes from 'prop-types';
...
+
+    this.state = {
+      modal: false,
+      value: ''
+    };
   }
 
+  handleModalOpen = () => {
+    this.setState({ modal: true });
+  };
```
Comment (sharonwarner) [inline client/app/hearings/components/HearingWorksheetIssues.jsx]: We should handle this through redux.

File: client/app/hearings/components/HearingWorksheetIssues.jsx (hunk 2 above)
Comment (nolawi) [reply client/app/hearings/components/HearingWorksheetIssues.jsx]: looks like its in base container...already.. can remove

**Comment by nolawi** [comment]
closing this till some refactor

**Pull request closed without merging**